app/UI/relay-detail.tsx

```diff
@@ -202,6 +202,7 @@ export class RelayInformationComponent extends Component<Props, State> {
                 const pubkey = PublicKey.FromString(info.pubkey);
                 if (pubkey instanceof Error) {
                     // todo make a UI
+                    console.log(info);
```

Comment:
In the future, maye the next PR, we need to use `zed` to runtime valid JSON shapes to prevent incorrect data from the server

Reply:
We get a `{pubkey: {hex: "xxx"}}` from `relayed`